ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
+If you are submitting a bug issue:
+- [ ] Please include your operating system type and version number, as well as your Python, scikit-learn, numpy, and scipy versions. This information can be found by runnning [this code snippet](https://gist.github.com/nelson-liu/108ad4a0571e43ac3d80).
```

Comment:
That's a good point! I can't find anything like it for numpy or scipy though...would it be best to just add the script to this repo and link to it?